### PR TITLE
Added ;;howtomusic and aliases, and a lot of info about the new commands to ;;music

### PR DESCRIPTION
**Pull request by RealKC**
;;music might have to be split in multiple messages soon.

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/command/util/MusicHelpCommand.java [hunk 1]
```diff
@@ -60,7 +60,21 @@
...
+            ";;gensokyoradio" +
+            "#Posts a special embed for gensokyoradio.net.\n" +
+            ";;forward\n" +
+            ";;fwd\n" +
+            "#Forwards to a future point in the current song using a specified amount of time.\n" +
+            ";;rewind\n" +
+            ";;rew\n" +
+            "#Rewinds to a previous point in  the current song using a specified amount of time.\n" +
+            ";;seek\n" +
+            "#Mves the song to a specific moment.\n" +
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/MusicHelpCommand.java]: There's a typo here

**Comment by RealKC** [comment]
ALL CHECKS HAVE FINALLY PASSED! YUS!

File: FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import net.dv8tion.jda.core.entities.*;
+import fredboat.util.BotConstants;
+
+public class HowToMusicCommand extends Command {
+    @Override
+    public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message message, String[] args) {
+
+        String description
+                = "**__How to invite FredBoat♪♪__**\n"
+                + "In order to use the bot's command you have to invite it first, to do so, you'll use [this](https://discordapp.com/oauth2/authorize?&client_id=184405253028970496&scope=bot) keep in my mind that you must have the Manage Server permission for this to work, if you do not have it, ask a mod about this.\n"
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java]: *commands

File: FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                + "**__How to play music with FredBoat♪♪__**\n"
+                + "To play music with the bot, you use the ;;play command, which can be used with an URL, allowing you to immediately play the requested song, or with the name of the song, searching it on youtube; you'll select the song with the ;;select n command where n is a number\n"
+                + "**__The ;;music command__**\n"
+                + "The ;;music command will post a list of all the music commands you can use. For quick reference, you can pin that message to a channel, if you have the Manage Messages permission, if you do not have this permission, ask a mod about this.";
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java]: Don't ask users to pin this message

File: FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+        MessageEmbed embed = new EmbedBuilder()
+                .setTitle("Helpful Tutorial for the music bot")
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java]: Title should be short, such as `Music Tutorial`. Don't make a claim of how useful it is.

File: FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java [hunk 5]
```diff
@@ -36,11 +36,11 @@ public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message m
...
+                + "In order to use the bot's commands you have to invite it first, to do so, you'll use [this](https://discordapp.com/oauth2/authorize?&client_id=184405253028970496&scope=bot) keep in mind that you must have the Manage Server permission for this to work, if you do not have it, ask a mod to invite the bot.\n"
                 + "**__How to play music with FredBoat♪♪__**\n"
-                + "To play music with the bot, you use the ;;play command, which can be used with an URL, allowing you to immediately play the requested song, or with the name of the song, searching it on youtube; you'll select the song with the ;;select n command where n is a number\n"
+                + "To play music with the bot, you use the ;;play command, which can be used with an URL, allowing you to immediately play the requested song, or with the name of the song, searching it on youtube; you'll select the song with the ;;select n or ;;play n commands where n is a number\n"
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java]: Format commands with code blocks. Eg \`;;play\` -> `;;play`

File: FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java [hunk 6]
```diff
@@ -43,7 +43,7 @@ public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message m
                 + "The ;;music command will post a list of all the music commands you can use. For quick reference, you can pin that message to a channel, if you have the Manage Messages permission, if you do not have this permission, ask a mod to do it.";
 
         MessageEmbed embed = new EmbedBuilder()
-                .setTitle("Helpful Tutorial for the music bot")
+                .setTitle("Music Tutorial")
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/HowToMusicCommand.java]: The title should link to `https://docs.frederikam.com`

File: FredBoat/src/main/java/fredboat/command/util/MusicHelpCommand.java [hunk 7]
```diff
@@ -60,7 +60,21 @@
...
+            ";;gensokyo\n" +
+            ";;gensokyoradio" +
+            "#Posts a special embed for gensokyoradio.net.\n" +
+            ";;forward\n" +
+            ";;fwd\n" +
+            "#Forwards to a future point in the current song using a specified amount of time.\n" +
+            ";;rewind\n" +
+            ";;rew\n" +
+            "#Rewinds to a previous point in  the current song using a specified amount of time.\n" +
+            ";;seek\n" +
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/util/MusicHelpCommand.java]: No argument is specified for ;;seek ;;rew and ;;fwd

**Comment by Frederikam** [comment]
Can you run an instance of the bot to see how it looks from Discord?

**Comment by Frederikam** [comment]
Closing this now as the upstream has changed too much for this to be merged.

**Pull request closed without merging**